src/core_plugins/kibana/public/visualize/wizard/step_1.html

```diff
@@ -15,9 +15,19 @@
 <div class="kuiViewContent kuiViewContent--constrainedWidth">
   <div class="kuiViewContentItem">
     <!-- Header -->
-    <h1 class="kuiTitle kuiVerticalRhythm kuiVerticalRhythm--medium">
-      Select visualization type
-    </h1>
+    <div class="visualizeViewContentHeader">
+      <h1 class="kuiTitle kuiVerticalRhythm kuiVerticalRhythm--medium">
+        Select visualization type
+      </h1>
+
+      <!--<div class="visualizeLabVisualizations" ng-if="labModeEnabled">-->
```

Comment:
Before merging, remove all that commented code and the CSS classes not needed anymore.